index.js

```diff
@@ -1460,13 +1460,74 @@ module.exports = function (log, indexesPath) {
     )
   }
 
+  function reindex(offset, cb) {
+    let seq = 0
+    if (offset === 0) {
+      seq = 0
+      offset = -1
+    } else if (offset !== -1) {
+      const { tarr } = indexes['seq']
+      let prevOffset = 0
+      for (const len = tarr.length; seq < len; ++seq)
```

Comment:
I feel a little unsafe with the for-loop not having tuborgs. Usually for-loops with no tuborgs have just one line under them, but this has 4. If someone wants to add one more line to this, it could become a footgun.